patch/0014-mlxsw-i2c-Allow-flexible-setting-of-I2C-transactions.patch

```diff
@@ -0,0 +1,409 @@
+From 1b96f4deebde7974332d9c498a559bb0931675c4 Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@mellanox.com>
+Date: Wed, 22 Jul 2020 20:26:51 +0300
+Subject: [PATCH net:backport] mlxsw: i2c: Allow flexible setting of I2C
+ transactions size
+
+Current implementation uses fixed size of I2C data transaction buffer.
```

Comment:
what is the upstream commit id?

Reply:
can you identify the upstream commit id for this patch?

Reply:
@lguohan commit details:
```
commit 3029a693beda5bff599feccc3beb9df1e141c786
Author: Vadim Pasternak <vadimp@mellanox.com>
Date:   Wed May 29 11:47:16 2019 +0300

    mlxsw: i2c: Allow flexible setting of I2C transactions size
    
    Current implementation uses fixed size of I2C data transaction buffer.
    Allow to set size of I2C transactions according to I2C physical adapter
    capability. For that purpose adapter read and write size is obtained
    from the I2C physical adapter and buffer size is set according to the
    minimum of these two values. If adapter does not provide such info,
    default buffer size is to be used.
    It allows to improve performance of I2C access to silicon when long
    size transactions are used.
    
    Signed-off-by: Vadim Pasternak <vadimp@mellanox.com>
    Acked-by: Jiri Pirko <jiri@mellanox.com>
    Signed-off-by: Ido Schimmel <idosch@mellanox.com>
    Signed-off-by: David S. Miller <davem@davemloft.net> 
```

Reply:
ok. can you add this original commit in the patch description?

Reply:
@lguohan i'll check with the maintainers and let you you know.